src/c++/perf_analyzer/model_parser.cc

```diff
@@ -169,6 +169,16 @@ ModelParser::InitTriton(
     response_cache_enabled_ = cache_itr->value["enable"].GetBool();
   }
 
+  // Check what the backend is:
+  const auto backend_config_itr = config.FindMember("backend");
```

Comment:
how does this handle ensemble models?
I have often seen pre and post processor steps show python as a backend but the main model had a backend that supported llms

Reply:
Do we have testing around this?